### DISCUSSION TITLE
Commodity vs Industry

### DISCUSSION THREAD

**Discussion by hajarpb** [USEPA/supply-chain-factors#14] [Q&A] [answered]
Hello,
I hope you are doing well,

In a few cases, we have a similar category in "commodity" and "industry" and EFs are slightly different. for example, in the "commodity" tab, we have "air transport" and in the industry tab, we have "air transportation".
What is the difference between these two? and which one we should use for scope 3 cat.1 and cat.2?

Many thanks

**Comment by MoLi7** [comment, marked as answer]
Hi @hajarpb 

Thanks for posting your question. In the Supply Chain GHG Factors webinar we hosted on March 10, @WesIngwersen specifically addressed this topic. You can watch the explanation [here](https://www.youtube.com/watch?v=pJ8gvZPdcgc&t=1029s).

Feel free to leave comments if you have further questions.

Thanks,
Mo

**Reply by hajarpb** [reply]
Thanks a lot, this is very helpful.

**Reply by MoLi7** [reply]
You are welcome! I think you (accidentally?) deleted the original question. Would you mind adding it back? That way others who have the same/similar question will be able to follow our conversation. Thank you!
